package.json

```diff
@@ -9,10 +12,16 @@
     "@capacitor/preferences": "^7.0.0",
     "@ionic/react": "^8.5.0",
     "@ionic/react-router": "^8.5.0",
+    "@types/google.maps": "^3.58.1",
```

Comment:
This should be removed since the app is no longer using Google Maps.

```suggestion
```

Reply:
Removed: https://github.com/ionic-team/ionic-react-conference-app/pull/133/commits/be76f7069d743d02a10152f61d15f7a234a60208